spec/features/1_user_signup_spec.rb

```diff
@@ -0,0 +1,33 @@
+feature 'User signup' do
+  scenario 'can create an account' do
+    expect { user_signup }.to change{ User.count }.by(1)
+    expect(page).to have_content 'Welcome, JohnnyD!'
+  end
+  scenario 'cannot sign up with mismatching passwords' do
+    expect { user_signup(password_confirmation: 'wrong passwords') }.not_to change{ User.count }
```

Comment:
Metrics/LineLength: Line is too long. [96/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)